Guide/file-storage.markdown

```diff
@@ -326,7 +326,7 @@ To store an image as a jpeg and reduce its quality use this:
 
 ```haskell
 let uploadLogo = uploadToStorageWithOptions $ def
-        { preprocess = applyImageMagick "jpg" "-resize '1024x1024^' -gravity north -extent 1024x1024 -quality 85% -strip"  }
+        { preprocess = applyImageMagick "jpg" ["-resize", "'1024x1024^'", "-gravity", "north", "-extent", "1024x1024", "-quality", "85%", "-strip"] }
```

Comment:
```suggestion
        { preprocess = applyImageMagick "jpg" ["-resize", "1024x1024^", "-gravity", "north", "-extent", "1024x1024", "-quality", "85%", "-strip"] }
```

The `'` is likely causing the error here

Here's an example of a production project that runs without error:

<img width="1068" alt="image" src="https://github.com/digitallyinduced/ihp/assets/2072185/01159590-8a18-4422-8cd0-e70b53db6238">

Reply:
Pushed a fix